modules/oe_authentication_eulogin_mock/src/EventSubscriber/CasMockServerSubscriber.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_authentication_eulogin_mock\EventSubscriber;
+
+use Drupal\cas_mock_server\Event\CasMockServerEvents;
+use Drupal\cas_mock_server\Event\CasMockServerResponseAlterEvent;
```

Comment:
I forgot to put credit to [Joinup implementation ](https://github.com/ec-europa/joinup-dev/blob/develop/web/modules/custom/joinup_cas_mock_server/src/EventSubscriber/JoinupCasMockServerSubscriber.php)

Reply:
Added comment to this class.